### ISSUE TITLE
Add issues reporting steps to docs

### ISSUE THREAD

**Issue by abitoprakash** [dequelabs/axe-core-maven-html#453]
Please add steps on how to setup violations reporting after analysing accessibility issues. I wanted to generate an html report with the results of each run.

**Comment by Zidious** [comment]
Hey @abitoprakash, 

We have a [AxeReporter](https://github.com/dequelabs/axe-core-maven-html/blob/c91e5ae030dec6b24d0755972c0fbb87a190909a/selenium/src/main/java/com/deque/html/axecore/selenium/AxeReporter.java#L32) class for writing axe-results to file if that's what you're looking for. Otherwise, we have extensive reporting capabilities with our axe-devtools-maven-html integrations: [docs.deque.com/devtools-html/4.0.0/en/java-generate-reports](https://docs.deque.com/devtools-html/4.0.0/en/java-generate-reports).

**Comment by Zidious** [comment]
Closing as complete. Feel free to reopen if you have any additional questions.
